From linux-usb-owner@vger.kernel.org Mon Feb 25 02:11:16 2008
From: tom.leiming@gmail.com
Date: Mon, 25 Feb 2008 18:07:28 +0800
Subject: USB: fix comment of struct usb_interface
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org, linux-usb@vger.kernel.org, Lei Ming <tom.leiming@gmail.com>
Message-ID: <1203934048-9718-1-git-send-email-tom.leiming@gmail.com>


From: Lei Ming <tom.leiming@gmail.com>

update the comment for the removed "driver" field  and  being
out-of-order of  @cur_altsetting and @num_altsetting.

Signed-off-by: Lei Ming <tom.leiming@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb.h |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/include/linux/usb.h
+++ b/include/linux/usb.h
@@ -94,10 +94,9 @@ enum usb_interface_condition {
  * @altsetting: array of interface structures, one for each alternate
  * 	setting that may be selected.  Each one includes a set of
  * 	endpoint configurations.  They will be in no particular order.
- * @num_altsetting: number of altsettings defined.
  * @cur_altsetting: the current altsetting.
+ * @num_altsetting: number of altsettings defined.
  * @intf_assoc: interface association descriptor
- * @driver: the USB driver that is bound to this interface.
  * @minor: the minor number assigned to this interface, if this
  *	interface is bound to a driver that uses the USB major number.
  *	If this interface does not use the USB major, this field should
